From 62d979db633fc70a519ac2bf7915b5d5381bd576 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Mon, 15 Oct 2012 13:44:20 -0700
Subject: Set owner/permission for /sys/power/autosleep

Set the owner and permission for /sys/power/autosleep
so Android can utilize opportunistic sleep, if early suspend
is not available.

Change-Id: I48b8d8c24c21c8ea7a7a10558c32ed0085607df4
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 rootdir/init.rc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index f7fe6f3..78c0305 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -293,6 +293,7 @@ on boot
     chown system system /sys/power/wakeup_count
     chown radio system /sys/power/wake_lock
     chown radio system /sys/power/wake_unlock
+    chmod 0660 /sys/power/autosleep
     chmod 0660 /sys/power/state
     chmod 0660 /sys/power/wake_lock
     chmod 0660 /sys/power/wake_unlock
-- 
1.7.4.1

